src/app/components/mnoe-api/admin/users.svc.coffee

```diff
@@ -69,6 +69,13 @@
   @sendSignupEmail = (email) ->
     MnoeAdminApiSvc.all('/users').doPOST({user: {email: email}}, 'signup_email')
 
+  @updateMember = (organization, member) ->
+    MnoeApiSvc.one('/organizations', organization.id).customPUT({member: member}, '/update_member')
+
+  @deleteMember = (organization, member) ->
+    data = { member: _.pick(member, "email") }
```

Comment:
if you only need the email, why not pass just the email instead of the whole object and then pick?